ui/js/services/pinboard.js

```diff
@@ -38,7 +38,7 @@ treeherder.factory('thPinboard', [
         };
 
         var saveBugs = function (job) {
-            _.forEach(relatedBugs, function (bug) {
```

Comment:
``relatedBugs`` here is actually an object, rather than an array.  So perhaps use ``Object.entries`` here.